charts/flux2/templates/crds/crds-helm-controller.yaml

```diff
@@ -0,0 +1,757 @@
+{{- if and .Values.helmcontroller.create}}
```

Comment:
for now it is ok but in the future I will like to have an option like this https://github.com/jetstack/cert-manager/blob/master/deploy/charts/cert-manager/values.yaml#L42

Reply:
good question here is if the controller pods can run without the crds - lets discuss this via an an issue and a follow up PR - thank